client/src/sections/Login/LoginModal.tsx

```diff
@@ -0,0 +1,40 @@
+import {Modal, TextField} from '@shopify/polaris';
+import React from 'react';
+
+export default function LoginModal(props: any) {
```

Comment:
We are using Typescript for a reason, so we avoid these cases where we are not typing. When I read this component how do I know what props this component should expect?

To type props, do this above the function definition (THIS IS JUST AN EXAMPLE!):
```
interface Props {
  email: string;
  password: number;
  active: boolean;
  handleChange: () => void;
}
```
and then on the function signature do this
```
export default function LoginModal({
  email,
  password,
  active,
  handleChange,
  ...etc
}: Props) {
  ...
}
```